single-audit/single_audit/settings/base.py

```diff
@@ -18,6 +18,7 @@
 
 ALLOWED_HOSTS = []
 
+SECRET_KEY = 'SECRET'
```

Comment:
Hmmmm — based on my understanding, this will set SECRET_KEY to "SECRET" in production, when we'd rather have a truly secret key in production. Say more about the reason for adding this line here?

Reply:
Will `production.py` not override/update the value from `base.py`?

I added this (back in) because manage.py was throwing errors about `SECRET_KEY` not having been set. Specifying ` --settings=single_audit.settings.development` on every run did eventually work, but I'd like to avoid having the default behavior be broken. 🤔

Reply:
Ah, gotcha. I was thinking of it less as "break default behavior" and more as "make sure it's never ambiguous which environment the app is being run in" ... though on reflection I suppose it's both! 😄 

I've been running 

`pipenv run ./manage.py runserver --settings=single_audit.settings.development` 


through my terminal history rather than typing it all out, so I guess that's why the extra verbosity hasn't been bothering me. Was also thinking of adding a `./run_locally` command that would hide away all that verbosity. But I'm fine deferring to whatever you think makes the most sense here.

Reply:
Got it. I want to think on this a little more — let's leave this as-is for now and revisit it in #87 (deployment to cloud.gov).